app/views/static/buttons/generic_button_component.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
This partial `app/views/static/buttons/generic_button_component.html.haml ` is the real deal. 
I mean, this is the one that will stay and the others will be deprecated gradually.

Separate PR for `generic_button_component` is not a bad idea, but I really need a form to go along with it. Also I need to demonstrate how the `generic_button_component` component can be embedded in the parent form component. So this is the PR to do it.